src/components/Navbar/Mobile/ResourcesDropdownMobile.tsx

```diff
@@ -0,0 +1,99 @@
+import { t, Trans } from '@lingui/macro'
+import { Collapse, Space } from 'antd'
+import CollapsePanel from 'antd/lib/collapse/CollapsePanel'
+import ExternalLink from 'components/shared/ExternalLink'
+import { useState } from 'react'
+
+import { DownOutlined, UpOutlined } from '@ant-design/icons'
+
+import { navMenuItemStyles } from '../navStyles'
+
+function ResourcesItem({
+  text,
+  route,
+  onClick,
+}: {
+  text: string
+  route?: string
+  onClick?: VoidFunction
+}) {
+  return (
+    <ExternalLink
+      className="nav-dropdown-item"
+      href={route}
+      onClick={onClick}
+      style={{ ...navMenuItemStyles, fontWeight: 400, marginTop: 0 }}
+    >
+      {text}
+    </ExternalLink>
+  )
+}
+
+export default function ResourcesDropdownMobile() {
+  const [activeKey, setActiveKey] = useState<0 | undefined>()
+  const iconSize = 12
+
+  // Close dropdown when clicking anywhere in the window
+  window.addEventListener('click', () => setActiveKey(undefined), false)
+
+  return (
+    <div className="resources-dropdown">
```

Comment:
**suggestion**: any chance we can use the antd `SubMenu` here instead? Would simplify this quite a bit I think.

https://ant.design/components/menu/

https://user-images.githubusercontent.com/94939382/158013586-183b160f-5371-4a49-9846-975e9fb3dac9.mp4

Reply:
Yep good idea I've switched to antd `SubMenu` and added its styles to antd-overrides